beetsplug/web/__init__.py

```diff
@@ -309,6 +309,7 @@ def __init__(self):
             'host': u'127.0.0.1',
             'port': 8337,
             'cors': '',
+            'subsonic': True
```

Comment:
If subsonic introduces a bunch of new dependencies, perhaps the default should be False?